src/styles/blocks/button.scss

```diff
@@ -0,0 +1,33 @@
+.button {
+  box-sizing: border-box;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: $c-general;
+  width: 288px;
+  height: 56px;
+  font-weight: 700;
+  color: $c-white;
+  text-decoration: none;
+  text-align: center;
+  border: 3px solid transparent;
+  border-radius: 25px;
+
+  cursor: pointer;
+
+  &--primary {
+    width: 100%;
+  }
+
+  &:hover {
+    background-color: $c-white;
+    color: $c-general;
+    border-color: $c-general;
+  }
+
+  &:active {
+    color: $c-white;
+    background-color: #111;
```

Comment:
Variable too

Reply:
but before i knew, that when we use only once some parameter, we shouldn't create different variable. or isn't true? :))